README.md

```diff
@@ -30,6 +30,22 @@ Currently we are targeting five platforms: OS X, Ubuntu Linux, iOS, Android, and
 
 ## platforms ##
 
+### Windows (command line) ###
+Downloading angle https://www.dropbox.com/s/vollhh52bjemeym/angle.7z?dl=0 and extract to `build` directory
```

Comment:
The Servo project has a fork of ANGLE with CMake build support: https://github.com/servo/angle